ui/main/src/app/modules/usercard/usercard.component.html

```diff
@@ -1,98 +1,165 @@
                 </label>
 
-                <label class="opfab-radio-button opfab-usercard-radio-button"> <span class="label-sev-compliant" translate> shared.severity.compliant </span>
-                    <input type="radio"  id="opfab-sev-compliant" name="severity" formControlName="severity" value="COMPLIANT">
+                <label class="opfab-radio-button opfab-usercard-radio-button"> <span class="label-sev-compliant"
+                        translate> shared.severity.compliant </span>
+                    <input type="radio" id="opfab-sev-compliant" name="severity" formControlName="severity"
+                        value="COMPLIANT" (change)="onSeverityChoiceChanged($event)">
                     <span class="opfab-radio-button-checkmark"></span>
                 </label>
 
-                <label class="opfab-radio-button opfab-usercard-radio-button"> <span class="label-sev-information" translate> shared.severity.information </span>
-                    <input type="radio"  id="opfab-sev-information" name="severity" formControlName="severity" value="INFORMATION">
+                <label class="opfab-radio-button opfab-usercard-radio-button"> <span class="label-sev-information"
+                        translate> shared.severity.information </span>
+                    <input type="radio" id="opfab-sev-information" name="severity" formControlName="severity"
+                        value="INFORMATION" (change)="onSeverityChoiceChanged($event)">
                     <span class="opfab-radio-button-checkmark"></span>
                 </label>
             </div>
 
-        </form>
-        <of-usercard-dates-form [datesFormInputData]="datesFormValue"
-                    (startDateFilterChange)="onStartDateChange()"
-                    (endDateFilterChange)="onEndDateChange()"
-                    (lttdFilterChange)="onLttdChange()"
-                    (expirationDateFilterChange)="onExpirationDateChange()"
-                    #datesForm> </of-usercard-dates-form>
-
 
-        <of-spinner *ngIf="isLoadingCardTemplate"></of-spinner>
-
-        <div [hidden]="isLoadingCardTemplate">
-            <div class="opfab-form-lighter" [innerHtml]="userCardTemplate"></div>
+            <div [hidden]="templateLoadingInProgress" class="justify-content-center"
+                style="display:flex;width:100%;align-content:center;flex-wrap: wrap;">
+                <div style="width:280px; margin: 30px" class="nopaddingrow opfab-input"
+                    id="opfab-usercard-input-startDate" hidden>
+                    <label for="opfab-usercard-startdate-choice-input" translate>userCard.filters.startDate</label>
+                    <input id="opfab-usercard-startdate-choice-input" type="datetime-local" formControlName="startDate"
+                        (change)="onStartDateChange()">
+                </div>
+
+                <div style="width: 280px; margin: 30px" class="nopaddingrow opfab-input"
```

Comment:
I think the css class nopaddingrow has been removed no ? I do not find it in any css/scss file